src/adapters/handlers/items.ts

```diff
@@ -1,66 +1,22 @@
-import { ethers } from 'ethers'
-import {
-  EmoteCategory,
-  EmotePlayMode,
-  GenderFilterOption,
-  ItemSortBy,
-  Network,
-  NFTCategory,
-  Rarity,
-  WearableCategory,
-} from '@dcl/schemas'
+import { ItemSortBy } from '@dcl/schemas'
 import { IHttpServerComponent } from '@well-known-components/interfaces'
 import { AppComponents, AuthenticatedContext } from '../../types'
 import { Params } from '../../logic/http/params'
-import { asJSON, HttpError } from '../../logic/http/response'
+import { HttpError, asJSON } from '../../logic/http/response'
+import { getItemsParams } from './utils'
 
 export function createItemsHandler(
   components: Pick<AppComponents, 'items'>
 ): IHttpServerComponent.IRequestHandler<AuthenticatedContext<'/items'>> {
   const { items } = components
   return async (context) => {
     const params = new Params(context.url.searchParams)
-
     const first = params.getNumber('first')
     const skip = params.getNumber('skip')
     const sortBy = params.getValue<ItemSortBy>('sortBy', ItemSortBy)
-    const category = params.getValue<NFTCategory>('category', NFTCategory)
-    const creator = params.getList('creator')
-    const isSoldOut = params.getBoolean('isSoldOut')
-    const isOnSale = params.getBoolean('isOnSale')
-    const search = params.getString('search')
-    const isWearableHead = params.getBoolean('isWearableHead')
-    const isWearableAccessory = params.getBoolean('isWearableAccessory')
-    const isWearableSmart = params.getBoolean('isWearableSmart')
-    const wearableCategory = params.getValue<WearableCategory>(
-      'wearableCategory',
-      WearableCategory
-    )
-    const rarities = params.getList<Rarity>('rarity', Rarity)
-    const wearableGenders = params.getList<GenderFilterOption>(
-      'wearableGender',
-      GenderFilterOption
-    )
-    const emoteCategory = params.getValue<EmoteCategory>(
-      'emoteCategory',
-      EmoteCategory
-    )
-    const emoteGenders = params.getList<GenderFilterOption>(
-      'emoteGender',
-      GenderFilterOption
-    )
-    const emotePlayMode = params.getList<EmotePlayMode>(
-      'emotePlayMode',
-      EmotePlayMode
-    )
+    const { contractAddresses, itemId, urns, ...restOfFilters } =
```

Comment:
Why is this necessary? For example, the `urns` are not used after deconstruction.

Reply:
They are used here: https://github.com/decentraland/nft-server/blob/master/src/adapters/handlers/items.ts#L70. Basically I'm leaving those 3 out of the rest so they can be used in that line